### PR TITLE
Redux state

### PR DESCRIPTION
**Pull request by Kacique**
Added state functionality to our app with Redux. Now redux handles the city name search query state.

### CODE REVIEW

File: api/cityNameSlice.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { createSlice } from "@reduxjs/toolkit";
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+export interface cityState {
+  value: string
+}
+
+const initialState: cityState = {
+  value: "",
```
Comment (kmausolf) [inline api/cityNameSlice.ts]: We may want to name this something more descriptive.  What does this value represent?  The city?  Zip code?  Let's try to make this obvious.

File: app/store.ts [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+  initialState,
+  reducers: {
+    getCity: (state, action: PayloadAction<string>) => {
+      state.value = action.payload;
+    },
+  },
+})
+
+export const { getCity } = cityNameSlice.actions;
+export default cityNameSlice.reducer;
```
Comment (kmausolf) [inline app/store.ts]: Don't we already have this in another file?  This should be safe to remove.

**Comment by kmausolf** [comment]
@lindakw Please change the base branch to `development`.  I'd recommend setting that to be the default branch in the repository settings.

File: screens/aqi_screen/aqiPage.tsx [hunk 3]
```diff
@@ -5,45 +5,63 @@ import { Ionicons, MaterialIcons } from "@expo/vector-icons";
...
+
 type AQIScreenProps = NativeStackScreenProps<StackParamList, "Main">;
 
 const Aqi: FC<AQIScreenProps> = (props) => {
+
+  let cityName = useSelector((store: RootState) => store.city.value)
+  const { data, error, isLoading } = useGetAQIQuery(cityName);
+
+  console.log(data);
+  console.log(error);
```
Comment (kmausolf) [inline screens/aqi_screen/aqiPage.tsx]: Leftover console logs, we should remove these.

**Review by kmausolf — CHANGES_REQUESTED** [review]
There's a duplication issue with the city slice here.  Once that's addressed and the base branch has been switched, I'll remove the hold to prevent any mishaps.

**Review by lindakw — APPROVED** [review]

**Review by kmausolf — APPROVED** [review]
Looks great!

File: api/cityNameSlice.ts (hunk 1 above)
Comment (lindakw) [reply api/cityNameSlice.ts]: value was changed to cityName

**Pull request merged**